### DISCUSSION TITLE
Adding badge to window title bar

### DISCUSSION THREAD

**Discussion by ahmedtalaat327** [lepoco/wpfui#1137] [Ideas]
I was wondering if it's possible to add a badge like this..?
![Untitled-1](https://github.com/lepoco/wpfui/assets/26097164/b9cde999-eee9-47ef-9a10-bf50c0118c18)

**Comment by rasyidf** [comment]
Yes, You can use header when adding additional controls like this

![image](https://github.com/lepoco/wpfui/assets/28984914/d47da827-193b-49f5-9c99-cc72720b77b8)


Just add header element inside the titlebar


```xaml
<ui:TitleBar
    x:Name="TitleBar"
    Title="{Binding ViewModel.ApplicationTitle}"
    Grid.Row="0"
    CloseWindowByDoubleClickOnIcon="True">
    <ui:TitleBar.Icon>
        <ui:ImageIcon Source="pack://application:,,,/Assets/wpfui-icon-256.png" />
    </ui:TitleBar.Icon>
    <ui:TitleBar.Header>
        <ui:Button Appearance="Caution">Beta</ui:Button>
    </ui:TitleBar.Header>
</ui:TitleBar>
```

**Comment by rasyidf** [comment]
perhaps you can mark it as answered @ahmedtalaat327
